tests/fixtures/tool_helpers.py

```
@@ -0,0 +1,62 @@
+                tool_calls=[{"id": call_id, "name": tool_name, "args": kwargs}],
+            ),
+        ]
+    }
+
+
+async def run_tool(
+    app,
+    tool_call_state: dict,
+    agent_context=None,
+    thread_id: str = "test",
+    **config_kwargs: Any,
```

Ruff (0.14.5)

Dynamically typed expressions (typing.Any) are disallowed in `**config_kwargs`

(ANN401)